File: Golang/788. Rotated Digits.go
import "strings"
func rotatedDigits(N int) int {
    count := 0
    for num := 1; num <= N; num++ {
        s := strconv.Itoa(num)
        if isrotatedDigits(s) {
            count++
        }
    }
    return count
}

func isrotatedDigits(s string) bool {
    if strings.Contains(s, "3") || strings.Contains(s, "4") || strings.Contains(s, "7") {
        return false
    } else if strings.Contains(s, "2") || strings.Contains(s, "5") || strings.Contains(s, "6") || strings.Contains(s, "9") {
        return true
    } else {
        return false
    }
}

File: Python3/788. Rotated Digits.py
class Solution:
    def rotatedDigits(self, N):
        """
        :type N: int
        :rtype: int
        """
        def isrotatedDigits(s):
            if '3' in s or '4' in s or '7' in s:
                return False
            elif '2' not in s and '5' not in s and '6' not in s and '9' not in s:
                return False
            else:
                return True
        count = 0
        for s in map(str, range(1, N+1)):
            if isrotatedDigits(s):
                count += 1
        return count
